modules/cactus-web/src/List/List.tsx

```diff
@@ -0,0 +1,112 @@
+import * as icons from '@repay/cactus-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { margin, MarginProps } from 'styled-system'
+
+import Flex from '../Flex/Flex'
+import Text from '../Text/Text'
+
+interface ListProps
+  extends MarginProps,
+    React.DetailedHTMLProps<React.HTMLAttributes<HTMLUListElement>, HTMLUListElement> {
+  divided?: boolean
+}
+
+interface ListItemProps
+  extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLLIElement>, HTMLLIElement> {
+  icon?: keyof typeof icons
+  header?: React.ReactNode
+  headerAs?: keyof JSX.IntrinsicElements | React.ComponentType<any>
+}
+
+const UL = styled.ul<{ $divided: boolean }>`
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  ${margin}
+
+  & > li > .item-flex > .item-inner-flex > ul {
+    margin-top: 8px;
+    margin-bottom: 8px;
+    margin-left: 24px;
+  }
+
+  ${(p) =>
+    p.$divided &&
+    `
+    li {
+      border-top: 1px solid ${p.theme.colors.lightContrast};
+    }
+    li:first-of-type {
+      border-top: none;
+    }
+  `}
+
+  .clickable {
+    cursor: pointer;
+  }
+`
+
+export const List = React.forwardRef<HTMLUListElement, ListProps>(
+  ({ children, divided = false, ...props }, ref) => (
+    <UL $divided={divided} {...props} ref={ref}>
+      {children}
+    </UL>
+  )
```

Comment:
So this is something I've been contemplating for a while, and I haven't really reached a conclusion. If you're not actually doing anything with the children or the internal structure of the element, is there any reason to pass them separately? Maybe it's more readable or just more expected? Which do you think looks better?
```suggestion
  ({ divided = false, ...props }, ref) => (<UL $divided={divided} {...props} ref={ref} />)
```

Reply:
Eh, I could go either way. Personally I think the way it's written now is a little easier to read just because you see the children explicitly being rendered as a child of `UL`, but like I said I could go either way.